scripts/quests/bastok/Beauty_And_The_Galka.lua

```diff
@@ -0,0 +1,151 @@
+-----------------------------------
+-- Beauty and the Galka
+-----------------------------------
+-- !addquest 1 1
+-----------------------------------
+require('scripts/globals/interaction/quest')
+require("scripts/globals/items")
+require("scripts/globals/keyitems")
+require("scripts/globals/npc_util")
+require('scripts/globals/quests')
+require('scripts/globals/zone')
+
+local portBastokIDs = require("scripts/zones/Port_Bastok/IDs")
+-----------------------------------
+local quest = Quest:new(xi.quest.log_id.BASTOK, xi.quest.id.bastok.BEAUTY_AND_THE_GALKA)
+
+quest.reward =
+{
+    fame = 75,
+    fameArea = xi.quest.fame_area.BASTOK,
+    item = xi.items.BRONZE_KNIFE
+}
+
+quest.sections =
+{
+    -- Quest Acceptance 1
+    {
+        check = function(player, status, vars)
+            return status == QUEST_AVAILABLE
+        end,
+
+        [xi.zone.PORT_BASTOK] =
+        {
+            ['Talib'] = quest:progressEvent(2),
+
+            onEventFinish =
+            {
+                [2] = function(player, csid, option, npc)
+                    if option == 1 then
+                        quest:setVar(player, 'Denied', 1)
+                        return
```

Comment:
The return is to exit the function before getting to quest:begin below if the option is 1.

Reply:
We could use an if else statement to correct that

Reply:
Of course, but "else" is a dirty word and I normally don't use them unless the function forcibly dictates needing it.  Does the early return affect anything in onEventFinish, since it doesn't _need_ a return  at all?

Reply:
we have such simple parameters here. if else won't be an issue. it's either one or the other essentially.